app/lib/io/zipUtil.js

```diff
@@ -12,7 +12,31 @@ define([
     var NotImplementedError = errors.NotImplemented;
 
     var unpack = function(async, zipData, io, targetPath){
-        throw new NotImplementedError('ZIP unpack method is not yet implemented');
+        if (async)
+            throw new NotImplementedError('Asynchronous ZIP unpack method is not yet implemented');
+
+        var zip = new JSZip(zipData)
+          , files = zip.files
+          , filename
+          , file
+          , absolute_path
+          , dir_abs_path
+          ;
+
+        console.log("unpacking files: ", files);
```

Comment:
You can also add an argument for a `logger` as MetapolatorProject has it at `this._log`

We shouldn't leave `console.log` in the code anymore, however. (I think ImportController is a very bad example for this.)